ibm_catalog.json

```diff
@@ -55,40 +70,44 @@
           },
           "iam_permissions": [
             {
+              "service_name": "iam-identity",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator",
                 "crn:v1:bluemix:public:iam-identity::::serviceRole:UserApiKeyCreator"
               ],
-              "service_name": "iam-identity",
-              "notes": "Allows IBM Cloud OpenShift to create the containers-kubernetes-key required by the service"
+              "notes": "Required to allow IBM Cloud OpenShift to create the containers-kubernetes-key needed by the service."
```

Comment:
Can we rephrase it like:
Required to create the containers-kubernetes-key needed by the OpenShift cluster on IBM Cloud.